src/gov/nist/core/LexerCore.java

```diff
@@ -44,6 +44,7 @@
     public static final int START = 2048;
     public static final int END = START + 2048;
     // IMPORTANT -- This should be < END
+    public static final int IPV6 = END - 4;
```

Comment:
apparently codes between start<->end are supposed to be keywords, that will be cached on globalSymbolTable map. I dont think IPv6 addresses can be treated as keywords,so you may need to assign a code above END. wdyt?